<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>360</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Your Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userEmailBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Your Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userPasswordBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Your Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icons/users.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="link">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Already Have an Account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
